### DISCUSSION TITLE
Remove Apple car play and Android auto

### DISCUSSION THREAD

**Discussion by andr3aLucch3si85** [Mr-MIBonk/M.I.B._More-Incredible-Bash#97] [Q&A] [answered]
Hi, I've patched the firmware to update FEC files and install new 2023 maps. Now I have the phone interface icon in the menu. The problem is that I have the charge only USB so I can't use ACP or AA. How can I remove phone interface function from firmware? I don't need that. Thanks

**Comment by harman-f** [comment, marked as answer]
You can deactivate it within the M.I.B GEM settings.

**Reply by andr3aLucch3si85** [reply]
Could you be more accurate? thanks

**Reply by harman-f** [reply]
Not possible in standard version.

open the Launcher-sda0.esd and look for:

```

###########################################################################################
#
# Main -> M.I.B. --> CarPlay -> Check Coding
#
###########################################################################################
screen check_coding carplay_activation

bitvalue
    value   per 0x0 0xC0020054
	range   7.7 7.7
	label   "Android Auto"
	entry 	"not active" 0
	entry 	"ON" 1

bitvalue
    value   per 0x0 0xC0020054
	range   8.0 8.0
	label   "CarPlay"
	entry 	"not active" 0
	entry 	"ON" 1

bitvalue
    value   per 0x0 0xC0020054
	range   8.5 8.5
	label   "MirrorLink"
	entry 	"not active" 0
	entry 	"ON" 1

bitvalue
    value   per 0 4101
	range   19.6 19.7
	label   "USB configuration"
	entry 	"off" 0
	entry 	"charge only" 1
	entry 	"full" 2
	entry 	"iPod - CarPlay & Android Auto" 3
```

change `bitvalue` to `bitselect`

afterwards you will be able to disabled CP and AA in this menu:
![image](https://user-images.githubusercontent.com/98130152/214658682-9a3cc85d-b075-4aba-8d66-2e2ca65b8194.png)

![image](https://user-images.githubusercontent.com/98130152/214658729-1b4210e5-adb2-46a8-9cdf-0853c3e7130c.png)

**Reply by andr3aLucch3si85** [reply]
It works! Thanks! Thanks!
